### DISCUSSION TITLE
New Balls

### DISCUSSION THREAD

**Discussion by Adacrop-The-Great** [Neverball/neverball#286] [Show and tell]
Hello everyone, I have made six new balls a few years ago. Here's a list of balls I made:

1. Baseball
2. Basketball
3. Invisible Ball
4. Jupiter
5. Neptune
6. Uranus

you can download them here
[adacrop-ball.zip](https://github.com/Neverball/neverball/files/8118059/adacrop-ball.zip)

**Comment by Adacrop-The-Great** [comment]
Please reply to this comment saying which one(s) is/are your favourite(s).

**Reply by DepresseDreamer** [reply]
My favorite is the Basketball one, to be honest.
